recipes/fastlinkingnumbers/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "fastlinkingnumbers" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/antequ/fastlinkingnumbers/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b3393bffb8669ab8214b1b86ae8b1d48abd70ba021134a3f2a5e5fcfd641e7d9
+  patches:
+    - CMakeLists.txt.patch
+    - test.sh.patch
+    - llvm.patch
+
+build:
+  number: 0
```

```suggestion
build:
  number: 0
  run_exports:
    - {{ pin_subpackage('libpng') }}
```
If you are going to include the headers and shared library, we need to include a run_export, so this package is included in the run requirements of any library that builds against it.